app/script/util/marked.js

```diff
@@ -1,32 +1,1073 @@
-/*
- * Wire
- * Copyright (C) 2017 Wire Swiss GmbH
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see http://www.gnu.org/licenses/.
- *
- */
-
-'use strict';
+/**
```

Comment:
Is this the plain marked.js code now? Then we should use it as a proper dependency.

Reply:
The changes are vast and this version now differs a lot from the marked main branch. It contains a lot of scenario specific tweaks and fixes. It wouldn't make much sense to maintain it a single file repo.

Reply:
Makes sense. As discussed f2f we will go ahead and merge this after the upcoming release in order to get as much internal test coverage as we can before we release this to the public.